autoconfigure/storage-stackdriver/src/main/java/zipkin/autoconfigure/storage/stackdriver/ZipkinStackdriverStorageAutoConfiguration.java

```diff
@@ -61,19 +58,19 @@ String projectId() {
     }
     try {
       return getDefaultProjectId();
-    } catch (IOException exception) {
+    } catch (Throwable t) {
```

Comment:
will need an import to keep this compiling :P

```suggestion
    } catch (Throwable t) {
      Call.propagateIfFatal(t);
```